spec/linter-pydocstyle-spec.coffee

```diff
@@ -1,4 +1,4 @@
-LinterPEP257 = require '../lib/init'
+LinterPydocstyle = require '../lib/init'
```

Comment:
Wasn't 100% clear what the capitalization of this should be... not sure if it's a class name or random variable or needs to be spec'd a certain way